docs/core/extensions/localization.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Localization in .NET
+description: Learn how to use the IStringLocalizer and IStringLocalizerFactory implementations with dependency injection in your .NET workloads.
```

Comment:
Seems like this should be expanded a bit to depict more general localization info.